src/ResourceManager/MachineLearning/Commands.MachineLearning.Test/TestData/GraphWebServiceDefinition_Prod.json

```diff
@@ -0,0 +1,190 @@
+﻿{
+  "properties": {
+    "title": "ServiceTitle",
+    "description": "This is some service description",
+    "readOnly": "false",
+    "keys": {
+      "primary": "S8ayf24PAsbQqy8Nm67uGf3MbOeorYKPY8QfXen2+JsBZV5Q0NFfk9tgC7mjJ3aCsUPY8yKCFVcI/sCcOlmkKw==",
```

Comment:
It is not allowed to specify real credentials used for a real service, even if this service is only created for a test and changed afterward.  Please find an alternate way to specify or generate this credential